coding-test/wonji/level-1/체육복-2.js

```diff
@@ -0,0 +1,37 @@
+function solution(n, lost, reserve) {
+    let haveNum = n - lost.length; 
+  	// 전체 인원에서 도난당한 사람을 빼준다.
+  	// haveNum은 체육복을 가진 사람의 숫자이다.
+
+  	// 여벌의 체육복을 지녔지만 도난당한 사람이 있다면 
+  	// 배열에서 지우고 haveNum 카운트를 올린다.
+    for (let i = 0; i < lost.length; i++) {
+        if (reserve.indexOf(lost[i]) !== -1) {
```

Comment:
값이 배열에 포함되어있는지의 여부는 includes 메서드를 활용하자.
```suggestion
        if (reserve.includes(lost[i])) {
```

Reply:
넵